app/views/admins/_user_entry.haml

```diff
@@ -1,70 +1,83 @@
+      .col-sm-5
+        .pull-right
+          %span.label.label-default
+            = t(".id")
+            = user.id
+          %span.label.label-info
+            = t(".guid")
+            = user.try(:person).try(:guid)
+    .row
+      .col-sm-8
         %dl.dl-horizontal
-          %dt= t('username')
+          %dt= t("username")
           %dd= user.username
-          %dt= t('.email')
+          %dt= t(".email")
           %dd= user.email
-          %dt= t('.diaspora_handle')
+          %dt= t(".diaspora_handle")
           %dd= user.person.diaspora_handle
-          %dt= t('.last_seen')
-          %dd= user.last_seen || t('.unknown')
-          -if user.invited_by.present?
-            %dt= t('.invite_token')
+          %dt= t(".last_seen")
+          %dd= user.last_seen || t(".unknown")
+          - if user.invited_by.present?
+            %dt= t(".invite_token")
             %dd= invite_code_url(user.invited_by.invitation_code)
-          %dt= t('.account_closed')
+          %dt= t(".account_closed")
           %dd
             - if user.person.closed_account
-              %span.label.label-warning= t('.yes')
+              %span.label.label-warning= t(".yes")
             - else
-              %span.label.label-success= t('.no')
-          %dt= t('.nsfw')
+              %span.label.label-success= t(".no")
+          %dt= t(".nsfw")
           %dd
             - if user.person.profile.nsfw
-              %span.label.label-warning= t('.yes')
+              %span.label.label-warning= t(".yes")
             - else
-              %span.label.label-success= t('.no')
+              %span.label.label-success= t(".no")
 
-        %h4= t('layouts.header.profile')
+        %h4= t("layouts.header.profile")
 
         %dl.dl-horizontal
-          %dt= t('people.profile_sidebar.born')
+          %dt= t("people.profile_sidebar.born")
           %dd= user.person.profile.birthday
-          %dt= t('people.profile_sidebar.gender')
+          %dt= t("people.profile_sidebar.gender")
           %dd= user.person.profile.gender
-          %dt= t('people.profile_sidebar.location')
+          %dt= t("people.profile_sidebar.location")
           %dd= user.person.profile.location
-          %dt= t('people.profile_sidebar.bio')
+          %dt= t("people.profile_sidebar.bio")
           %dd= user.person.profile.bio
+
+      .col-sm-4
+        = link_to t("admins.user_search.view_profile"),
+          person_path(user.person),
+          class: "btn btn-default btn-block"
+        = link_to t("admins.user_search.add_invites"),
+          add_invites_path(user.invitation_code),
+          class: "btn btn-info btn-block"
+        - unless user.person.closed_account
+          = link_to t("admins.user_search.close_account"),
+            admin_close_account_path(user),
+            method: :post, data: {confirm: t("admins.user_search.are_you_sure")},
+            class: "btn btn-danger btn-block"
+
+        - unless user.closed_account?
```

Comment:
There is no difference, `user.closed_account?` delegates to  `user.person.closed_account`: https://github.com/diaspora/diaspora/blob/7eb3dabb444d99cf51e672e065284f9aa23628e9/app/models/user.rb#L514-L514